src/section.js

```diff
@@ -40,20 +40,20 @@ class Section {
 	 * @param  {method} _request a request method to use for loading
 	 * @return {document} a promise with the xml document
 	 */
-	load(_request){
-		var request = _request || this.request || require("./utils/request");
+	load(){
+		var request = require("./utils/request");
 		var loading = new defer();
 		var loaded = loading.promise;
 
 		if(this.contents) {
 			loading.resolve(this.contents);
 		} else {
-			request(this.url)
-				.then(function(xml){
+			request(this.url, null, null, null, true)
+				.then(function(response){
 					// var directory = new Url(this.url).directory;
 
-					this.document = xml;
-					this.contents = xml.documentElement;
+					this.document = response.responseXML;
```

Comment:
Lets address this response issue in the request function.
It should still return xml here.